outputs.tf

```diff
@@ -212,3 +212,10 @@ output "bastion_ingress_sgid" {
   value       = data.terraform_remote_state.bastion.outputs.ingress_from_bastion_sg_id
 }
 
+# IAM ROLES/PROFILES
+
+# SSM Profile name
```

Comment:
Since this output is part of the "Shared Global" outputs, it should be organized with the other similar outputs from "global" (maybe around line 164?).